### PR TITLE
Post for Opera 46

### PR DESCRIPTION
**Pull request by zcorpan**

### CODE REVIEW

File: src/blog/_posts/2017-06-22-opera-46.md [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+Opera 46 (based on Chromium 59) for [Mac, Windows,
+Linux](https://www.opera.com/computer) is out! To find out what's new
+for users, see our [Desktop](https://www.opera.com/blogs/desktop/)
+blog. Here's what it means for web developers.
+
+## Animated PNG
+
+Opera now supports animated PNG, or APNG for short. APNG is a file format that works similarly to
+GIF. The difference is that APNG is smaller and supports both 24-bit images and 8-bit transparency.
+It has become quite popular recently, particularly since Apple [adopted the APNG file format for iOS
```
Comment (fsoder) [inline src/blog/_posts/2017-06-22-opera-46.md]: "...for the iOS..."

(And should the link perhaps only cover "the iOS 10 iMessage apps"? Just a thought.)

File: src/blog/_posts/2017-06-22-opera-46.md [hunk 2]
```diff
@@ -0,0 +1,84 @@
...
+
+Watch the [APNG demo](http://littlesvr.ca/apng/gif_apng_webp1.html) and
+[learn more about this format](https://en.wikipedia.org/wiki/APNG).
+
+## SVG favicons
+
+This was actually added in Opera 44, but we missed to write about it!
+
+This might still be a bit rough in the edges, but in general SVG favicons should now work in Opera!
+(But [not yet in Chromium](https://bugs.chromium.org/p/chromium/issues/detail?id=294179).)
```
Comment (fsoder) [inline src/blog/_posts/2017-06-22-opera-46.md]: Drop the "But" - so just "Not yet..."

File: src/blog/_posts/2017-06-22-opera-46.md [hunk 3]
```diff
@@ -0,0 +1,84 @@
...
+* Following [removal](https://github.com/w3c/csswg-drafts/commit/2078b46218f7462735bb0b5107c9a3e84fb4c4b1) from the [Media Queries](https://www.w3.org/TR/css3-mediaqueries/) spec, support for `hover: on-demand` and `any-hover: on-demand` media queries have been removed.
+* To better align with spec and help avoid race conditions, [`decodeAudioData`](https://webaudio.github.io/web-audio-api/#widl-BaseAudioContext-decodeAudioData-Promise-AudioBuffer--ArrayBuffer-audioData-DecodeSuccessCallback-successCallback-DecodeErrorCallback-errorCallback) now detaches the given ArrayBuffer before decoding, removing all content from the object and making it unable to be reused or examined.
+* The `-internal-media-controls-cast-button` CSS selector has been removed in favor of the [Remote Playback API](https://w3c.github.io/remote-playback/).
+* The `-internal-media-controls-text-track-list*` CSS selectors have been removed in favor of custom-built video controls.
+* The `SVGTests.requiredFeatures` attribute has been deprecated following its removal from the [spec](https://github.com/w3c/svgwg/commit/9a30d01f6410dc516c5f874d71e957230a3448cd).
```
Comment (fsoder) [inline src/blog/_posts/2017-06-22-opera-46.md]: s/deprecated/removed/ (deprecation was in Opera 41/Chromium 54, https://www.chromestatus.com/features/5720709590417408)

**Comment by zcorpan** [comment]
Fixed

**Comment by lionux1988** [comment]
*notions with results. Google Inc. lionux1988@Android.Com*

**Pull request merged**